src/Layout/QueryParamContext.tsx

```diff
@@ -10,6 +10,7 @@ import { THANK_YOU } from "src/Utils/donationStepConstants";
 import { PaymentSetupProps } from "src/Common/Types";
 import { useAuth0 } from "@auth0/auth0-react";
 import { validateToken } from "../Utils/tokenActions";
+import allLocales from "../../i18n.json";
```

Comment:
I wonder if using `next-i18next.config.js` or `i18n.json` is more correct here as I do not know anymore if `i18n.json` is still used by our Next.js application.